### ISSUE TITLE
Feature Request: SpamFilter Plugin (ideally bayesian)

### ISSUE THREAD

**Issue by pinkerltm** [Foundry376/Mailspring#2054]
Years ago there was a Plugin for MS Outlook (yes, until 2010 Outlook was pretty usable for me) called "SpamBayes" (http://spambayes.sourceforge.net/)... it is written in Python and utilizes Bayesian Regression to classify mails as Spam or Not Spam. It wasn't only extraordinary because of the classification algorithm, but also because of the perfect UI Integration which allowed to easily supervise the algorithm while learning until it was  doing well (after manually selecting 10 to 20 Mails manually I hardlly had any misclassification which I could then use to retrain the algorithm)

Long story short....it was working like a charm.

I would love to see a plugin like that for mailspring again which really would be a major feature I missed the last 10 years or so in any mail client I tried (Outlook since 2013, Thunderbird, Evolution, Gmail, ...). If you already know a solution like that please share.

**Comment by CodeMouse92** [comment]
We are in the process of migrating issues to Discourse, which can better facilitate discussion and discovery, and so GitHub Issues can focus on issues that are confirmed and slated for resolution in the near term. [Learn more about the changes here.](https://community.getmailspring.com/t/mailspring-community-transition/38)

This appears to be related to an issue we've already migrated to Discourse:

https://community.getmailspring.com/t/adaptive-junk-filtering/834

Please consider joining that community and continuing the discussion there! **Votes on the feature suggestion on Discourse will increase the likelihood we implement this.**

@pinkerltm: if you join and reply to the issue, the moderators can make an effort to reassign your posts to you, so you get the credit for them.

We're closing and locking the issue here as part of this migration. Rest assured, this doesn't mean the issue is being discarded or ignored.

We hope to see you on Discourse soon!

-The Mailspring Team
